doc/02-installation.md

```diff
@@ -209,23 +209,10 @@ openSUSE:
 ```
 rpm --import https://packages.icinga.com/icinga.key
 
-zypper ar https://packages.icinga.com/openSUSE/ICINGA-release.repo
+zypper ar https://packages.icinga.com/openSUSE/ICINGA-testing.repo
 zypper ref
 ```
 
-#### Alpine Linux Repositories <a id="package-repositories-alpine"></a>
-
-```
-echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories
-echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories
-apk update
-```
-
-The example provided assumes that you are running Alpine edge, which is the -dev branch and is a rolling release.
-If you are using a stable version please "pin" the edge repository on the latest Icinga 2 package version.
-In order to correctly manage your repository, please follow
-[these instructions](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management)
-
```

Comment:
Why did you drop this one _right now_?

Reply:
Because we don't have any repositories that contain our RC releases on Alpine.